modules/core/04-channel/types/msgs.go

```diff
@@ -61,6 +61,8 @@ func (msg MsgChannelOpenInit) GetSigners() []sdk.AccAddress {
 var _ sdk.Msg = &MsgChannelOpenTry{}
 
 // NewMsgChannelOpenTry creates a new MsgChannelOpenTry instance
+// The version string is deprecated and will be ignored by core IBC.
```

Comment:
Could you also note the deprecation in the proto file?

Reply:
See `deprecated` section of:

https://developers.google.com/protocol-buffers/docs/proto3#options

Reply:
This isn't possible. `MsgChannelOpenTry` contains a `Channel` type, not individual fields for each value within the channel. Trying to deprecate it within `MsgChannelOpenTry` would deprecate field within the channel struct

Reply:
Nice, makes sense